package.json

```diff
@@ -42,7 +42,6 @@
   ],
   "dependencies": {
```

Comment:
The generator itself will not have to depend on `spellbook` unless we switch the build system to use it. The only problem with that would be the lack of support for running mocha tests, and even if we wanted to switch to qunit there is no good way to run qunit tests in node.